lib/api/resources.ts

```diff
@@ -852,6 +852,9 @@ export const RESOURCES = {
   graphql: {
     path: '/api/v1/graphql',
   },
+  block_countdown: {
```

Comment:
maybe it shouldn't be "block countdown" or we can specify query parameters here?

Reply:
It is only for typescript purposes. All resources in the old API share the same path, but this way we can define response types more granularly. Since API is deprecated I don't want to add more logic here now.